.github/label-commenter-config.yml

```diff
@@ -1,5 +1,5 @@
 comment:
-  header: "Please note the following requirement:"
+  # header: "Please note the following requirement:"
```

Comment:
@chetak123: It's just a `header` reminder, pl don't comment it out, also that's by @leecalcote itself!

He always has some plans while doing such stuff!

@leecalcote: Any insights?